client/galaxy/scripts/mvc/form/form-section.js

```diff
@@ -27,6 +31,14 @@ var View = Backbone.View.extend({
         });
     },
 
+    /** Only call render function if view has not been rendered before */
```

Comment:
Do we ever need to rerender based on changes in the model?

Reply:
Maybe, but then the model changes should just trigger `render` ... which is my unverified thought. Do you have an idea how to check this ?

Reply:
@mvdbeek Not off the top of my head; I haven't looked at the tool form logic here in a long time.  This was just the only thing I thought of that might be a concern.  @guerler would probably know if it's something to worry about; hopefully it's not.

Reply:
@mvdbeek I spent some time testing this, and I'm not worried about it.  Nice improvement here.